### PR TITLE
fix: correct javascript error

### PR DESCRIPTION
**Pull request by mica16**
Major error: `getActiveKeysAtHeight` is not a function but an instance method.

fixed, + some auto-indentation with IntelliJ.

### CODE REVIEW

File: src/app/factom-identity-manager.js [hunk 1]
```diff
@@ -101,7 +101,13 @@ class FactomIdentityManager {
      * @returns {{ txId: string, repeatedCommit: boolean, chainId: string, entryHash: string }} - Info about the Entry insertion.
      */
     async replaceIdentityKey(identityChainId, keys, ecAddress) {
-        return replaceIdentityKey(this.cli, identityChainId, keys, ecAddress);
+        return replaceIdentityKey(
+            this.cli,
```
Comment (PaulBernier) [inline src/app/factom-identity-manager.js]: this param is not necessary anymore

File: src/app/identity-management.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ async function replaceIdentityKey(cli, identityChainId, keys, ecAddress) {
         secret: keys.signingSecretIdKey
     };
 
-    const activeKeys = await getActiveKeysAtHeight(cli, identityChainId);
+    const activeKeys = await identityRetriever.getActiveKeysAtHeight(cli, identityChainId);
```
Comment (PaulBernier) [inline src/app/identity-management.js]: getActiveKeysAtHeight doesn't take `cli` as its first argument, just `identityChainId`

Comment (PaulBernier) [reply src/app/identity-management.js]: Also the incorrect import of `getActiveKeysAtHeight` at the top of this file can be removed

**Review by PaulBernier — CHANGES_REQUESTED** [review]
Oops that's bad, seems this part didn't get refactored properly, thanks for catching that.

**Comment by mica16** [comment]
Right, it was due to wrong copy paste between the project I'm working on and this fork.

You say that cli is not used any more within replaceIdentityKey but this one makes a final call using it: return cli.add(entry, ecAddress);

May you valid this PR and merge it if it is ASAP?

Thank you!

**Review by PaulBernier — APPROVED** [review]

**Comment by PaulBernier** [comment]
Published on npm with version 0.2.1. Thanks!

**Comment by mica16** [comment]
Thanks :)

**Pull request merged**